src/app/data/testimonials.jsx

```diff
@@ -0,0 +1,21 @@
+const testimonials = [
+  {
+    name: "Emily Johnson",
+    location: "USA, California",
+    rating: 5,
+    text: "Damien's photography doesn't just capture moments; it captures emotions. His work is simply mesmerizing.",
+  },
+  {
+    name: "John Smith",
+    location: "USA, California",
+    rating: 5,
+    text: "Damien has an incredible talent for making every event feel effortless, and the results speak for themselves.",
+  },
+  {
+    name: "Samantha Davis",
+    location: "USA, California",
+    rating: 5,
+    text: "I was blown away by Damien's ability to capture the essence of our wedding day. His photographs are our cherished memories.",
+  },
+];
+export default testimonials;
```

Comment:
for now I'd move it to Testimonials.jsx
also this file should've been named testimonials.js as it doesn't have a jsx syntax, but it doesn't matter now anyway